pkg/build/controller/common/util.go

```diff
@@ -134,3 +140,48 @@ func HasBuildPodNameAnnotation(build *buildapi.Build) bool {
 	_, hasAnnotation := build.Annotations[buildapi.BuildPodNameAnnotation]
 	return hasAnnotation
 }
+
+// ProcessValueFrom supports the k8s pod syntax of allowing "valueFrom" in addition
```

Comment:
A little more godoc might be nice - say something about what the function actually does